cron/auth02mozdef.py

```diff
@@ -224,37 +245,50 @@ def process_msg(mozmsg, msg):
             # Update a rule, add a site, update a site, etc
             details["description"] = msg.description
     except KeyError:
-        details["description"] = ""
+        pass
```

Comment:
Just so I understand the logic here, a KeyError would default to the loaded value becoming an empty string anyway? Or would it default to the value not existing at all?

If the second, have we verified that there are no alerts or other code that requires a details.description?

Reply:
This was written this way due to the way the summary was originally built, description needed to exist as they did no checks to determine if it was populated. With the summary change, this is no longer a required field when empty.